apps/frontend/src/components/SideBarRight/SideBarRight.tsx

```diff
@@ -22,6 +22,10 @@ const PROPERTY_COMPONENTS_MAP: Record<string, React.FunctionComponent<any>> = {
     SIZE: SizeProperty
 };
 
+function isComponentProp(prop: object): prop is ComponentProp {
```

Comment:
I don't understand what you mean

Reply:
It's called a type guard.
1. the function is called with `prop` of type `object`.
2. If the condition is satisfied, it returns `true`, and change the type of `prop` to `ComponentProp`
3. Else, it only returns `false` without any change